spec/models/capabilities/scopes/default_spec.rb

```diff
@@ -367,6 +369,7 @@
               .map { |_, v| v[:actions].map { |vk, vv| vv.map { |vvv| item.call(vk, vvv, v[:global], v[:module_name]) } } }
               .flatten(2)
               .compact
+              .uniq { |v| v.join(",") }
```

Comment:
This is needed, because `show` is allowed by the create and the manage user roles. So they will be added twice. This cleans it up. The `join` is needed because `uniq` uses the `eql?` methods and two arrays are not equal based on their contents